### PR TITLE
Health check | playbook names fix + Auto close option in main playbook

### PR DESCRIPTION
**Pull request by altmannyarden**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://github.com/demisto/etc/issues/33621

## Description
Changed the playbook names to incidents intend of playbooks. 
Add auto close option to the main playbook inputs.

## Minimum version of Demisto
- [ ] 5.0.0
- [ ] 5.5.0
- [x] 6.0.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/IntegrationsAndIncidentsHealthCheck/Playbooks/playbook-JOB_-_Integrations_and_Playbooks_Health_Check.yml [hunk 1]
```diff
@@ -1062,6 +1062,119 @@ tasks:
...
     quietmode: 0
+  "37":
+    id: "37"
+    taskid: 37c324dd-f456-4b40-8bd6-7742369ee767
+    type: condition
+    task:
+      id: 37c324dd-f456-4b40-8bd6-7742369ee767
+      version: -1
+      name: Automatically close investigation?
+      description: Checks the "AutoCloseInvestigation" playbook input equel to 'True'.
```
Comment (michalgold) [inline Packs/IntegrationsAndIncidentsHealthCheck/Playbooks/playbook-JOB_-_Integrations_and_Playbooks_Health_Check.yml:1073]: Checks if the "AutoCloseInvestigation" playbook input is 'True'.

**Review by michalgold — APPROVED** [review]

**Review by anara123 — COMMENTED** [review]
Add 
```
[file:JOB - Integrations and Playbooks Health Check]
ignore=BA101
```
in .pack-ignore for each playbook that changed name

**Comment by altmannyarden** [comment]
> Add
> 
> ```
> [file:JOB - Integrations and Playbooks Health Check]
> ignore=BA101
> ```
> 
> in .pack-ignore for each playbook that changed name
@anara123 Done

File: Packs/IntegrationsAndIncidentsHealthCheck/.pack-ignore [hunk 2]
```diff
@@ -1,2 +1,8 @@
 [file:playbook-JOB_-_Integrations_and_Playbooks_Health_Check.yml]
 ignore=PB105
+[file:JOB - Integrations and Playbooks Health Check]
```
Comment (anara123) [inline Packs/IntegrationsAndIncidentsHealthCheck/.pack-ignore:3]: you have to provide file name (file path)
like in the first line

Comment (altmannyarden) [reply Packs/IntegrationsAndIncidentsHealthCheck/.pack-ignore]: Done

**Pull request merged**